### PR TITLE
[Refactor] Extract accessibility id out of LocalizationKey

### PR DESCRIPTION
**Pull request by jimchou-dev**
## Purpose

* Extract accessibility id out of LocalizationKey
* Rename accessibility label that are accessibility id
* Remove LocalizationKey reference from demo app

## Does this introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ x ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
Verify that the following are valid
* All unit test and UI test are still passing

### CODE REVIEW

File: AzureCommunicationUI/AzureCommunicationUI/Utilities/AccessibilityIdentifier.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    case leaveCallAccessibilityID = "AzureCommunicationUICalling.CallingView.Overlay.LeaveCall.AccessibilityID"
+    case joinCallAccessibilityID = "AzureCommunicationUICalling.SetupView.Button.JoinCall.AccessibilityID"
+    case toggleVideoAccessibilityID = "AzureCommunicationUICalling.SetupView.Button.Video.AccessibilityID"
+    case togglMicAccessibilityID = "AzureCommunicationUICalling.SetupView.Button.Mic.AccessibilityID"
+    case videoAccessibilityID = "AzureCommunicationUICalling.CallingView.ControlButton.Video.AccessibilityID"
+    case micAccessibilityID = "AzureCommunicationUICalling.CallingView.ControlButton.Microphone.AccessibilityID"
+    case audioDeviceAccessibilityID =
+            "AzureCommunicationUICalling.CallingView.ControlButton.AudioDevice.AccessibilityID"
+    case hangupAccessibilityID = "AzureCommunicationUICalling.CallingView.ControlButton.HangUp.AccessibilityID"
+    case cancelAccessibilityID = "AzureCommunicationUICalling.CallingView.Overlay.Cancel.AccssibilityID"
```
Comment (vhuseinova-msft) [inline AzureCommunicationUI/AzureCommunicationUI/Utilities/AccessibilityIdentifier.swift:18]: typo `AccssibilityID`

File: AzureCommunicationUI/AzureCommunicationUIDemoApp/AppConfig.xcconfig [hunk 2]
```diff
@@ -4,4 +4,8 @@
 //
 #include? "EnvConfig.xcconfig"
 
-
```
Comment (vhuseinova-msft) [inline AzureCommunicationUI/AzureCommunicationUIDemoApp/AppConfig.xcconfig:7]: This info should be removed.

**Review by vhuseinova-msft — APPROVED** [review]
Looks good, tested!

**Review by palatter — APPROVED** [review]
Looks good!

**Pull request merged**